### PR TITLE
feat : Added the appointment API

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/assesment/views.py [hunk 1]
```diff
@@ -14,3 +14,14 @@ class AvalabilityViewSet(viewsets.ModelViewSet):
...
     def perform_create(self, serializer):
         serializer.save(doctor=self.request.user.doctor)
+
+
+class AppointmentViewSet(viewsets.ModelViewSet):
+    queryset = Appointment.objects.all()
+    serializer_class = AppointmentSerializer
+    permission_classes = [permissions.IsAuthenticated,]
+
+    def perform_create(self, serializer):
```
Comment (Raam124) [inline apps/assesment/views.py:24]: add validation to check if the requested availability is already added to appointment or not

Comment (Raam124) [reply apps/assesment/views.py:24]: if added return a meaningful error message

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/assesment/views.py:25]: Noted .

Comment (Raam124) [reply apps/assesment/views.py:25]: why is this still open ?

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py
Comment (Raam124) [inline apps/assesment/views.py:1]: need to appointment update API to update the status 
Also need the appointment to delete API to update the status to cancel

**Review by Raam124 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**